BE/src/main/java/louie/dong/airbnb/accommodation/dto/AccommodationDetailResponse.java

```diff
@@ -20,12 +23,12 @@ public class AccommodationDetailResponse {
 	private RoomInformation roomInformation;
 	private String description;
 	private int price;
-	private boolean wishlist;
+	private boolean isWish;
 
 	public AccommodationDetailResponse(Accommodation accommodation, boolean wish) {
 		this.id = accommodation.getId();
 		this.name = accommodation.getName();
-		this.imageUrl = accommodation.getAccommodationImages().get(0).getImageUrl();
+		this.imageUrl = getImageUrl(accommodation.getAccommodationImages());
```

Comment:
제가 코멘트 답변을 못드렸네요.
대표 이미지 찾는것도 숙소에 메소드를 선언해 주시면 좋을것 같습니다.

```suggestion
		this.imageUrl = accommodation.getThumbnail();
```